tests/test_parsl_flux_mpi_hello.py

```diff
@@ -85,6 +85,10 @@ def test_flux_resource_list(load_config):
                       stderr=('parsl_flux_resource_list.err', 'w'),
                       env=load_config["environment"]).result()
     assert r == 0
+
+    pattern = re.compile(r'.*', re.DOTALL)
```

Comment:
I am still working on the regex expression, so far I just wanted to confirm that my idea would work. `.*` just matches any expression and adding the `re.DOTALL` matches newline characters as well. Currently it passes when compared to something line:
```
     STATE NNODES   NCORES    NGPUS NODELIST
      free      3       47        0 slurmnode[1-3]
 allocated      1        1        0 slurmnode3
      down      0        0        0 
```

I'll be adding a line by line match to the current regex expression and it will look similar to:
```
pattern = re.compile(
    r'<insert regex expression of column 1>\n'
    r'<insert regex expression of column 2>\n'
    r'<insert regex expression of column 3>\n'
    r'<insert regex expression of column 4>\n', 
re.DOTALL)
```